x-pack/legacy/plugins/integrations_manager/server/integrations/get.ts

```diff
@@ -45,7 +47,7 @@ export async function getIntegrationInfo(options: {
 
   // add properties that aren't (or aren't yet) on Registry response
   const updated = Object.assign({}, item, {
-    title: item.description.split(' ')[0],
+    title: item.title || item.description.split(' ')[0],
```

Comment:
Should the fallback for title be `name`. I think I misread that code somehow, because so far it worked.

Reply:
`name` would give something like "mysql" which is all lowercase so not suitable for a title (e.g. card or page header). We could uppercase the first letter ("Mysql"). 

It's worked so far because the data coincidentally has a properly formatted value as the first word. I used it to make the components look realistic, but it would fail badly if the description became "A battle-tested database..." leaving title of "A"

I'll update to titlecase `name` and ensure a better failure mode.